release.md

```diff
@@ -1,14 +1,21 @@
-# Netclient v0.18.8
+# Netclient v0.19.0
 
 ## Whats New
-- Unprivilaged GUI app
+
+- Unprivilged GUI
+    netclient-gui runs without root/administrator permissions; requires netclient daemon 
+- TURN
+    nodes behind asymetrical/double NAT will use TURN to communicate
+- dependency updates
 
 ## What's Fixed
-- 
-- 
-- 
-- 
-- 
+- unbiased random string
+- CI updates
+- creation of explicit server/peer routes for internet gateways
+- firewall detection
+- nftables forwarding
+
+
 ## known issues
 - Incorrect metrics against ext clients
 - Host ListenPorts set to 0 after migration from 0.17.1 -> 0.18.6
```

Comment:
should include the GUI bug mentioned in NET-28

Reply:
added